app/uk/gov/hmrc/timetopay/arrangement/model/model.scala

```diff
@@ -184,3 +184,112 @@ object modelFormat {
   implicit val desSubmissionRequestFormat: OFormat[DesSubmissionRequest] = Json.format[DesSubmissionRequest]
 
 }
+
+case class AnonymousTTPArrangement(
+    _id:                  String,
+    createdOn:            LocalDateTime,
+    paymentPlanReference: String,
+    directDebitReference: String,
+    taxpayer:             AnonymousTaxpayer,
+    bankDetails:          BankDetails,
+    schedule:             PaymentSchedule,
+    desArrangement:       Option[AnonymousDesSubmissionRequest])
+
+object AnonymousTTPArrangement {
+  implicit val localDateTimeFormat: Format[LocalDateTime] = MongoJavatimeFormats.localDateTimeFormat
+
+  implicit val format: OFormat[AnonymousTTPArrangement] = Json.format[AnonymousTTPArrangement]
+
+  implicit def makeArrangementAnonymous(ttpArrangement: TTPArrangement): AnonymousTTPArrangement = {
```

Comment:
I'm not sure about using an implicit conversion here. They are frowned upon as they can have unwanted and hard-to-trace effects. Better just to have an 'applyd method and use explicitly where it is needed.

```scala
def apply(ttpArrangement: TTPArrangement): AnonymousTTPArrangement = ...
```